Scripts/DCS-BIOS/doc/json/AircraftAliases.json

```diff
@@ -64,6 +64,7 @@
                "MB-339A": [ "CommonData", "MB-339" ],
             "MB-339APAN": [ "CommonData", "MB-339" ],
              "MB-339PAN": [ "CommonData", "FC3" ],
+                "MH-60R": [ "MH-60R" ],
```

Comment:
Should also have commondata. I'm guessing this needs to be added to AircraftList.lua as well.